### DISCUSSION TITLE
Welcome, SebastienXia!

### DISCUSSION THREAD

**Discussion by SebastienXia** [ParanoidUser/yolo#558] [Subscribe]
Ho yo can I with Contributor permissions to make changes to the repository ☠️

**Comment by ParanoidUser** [comment]
Ahoy, @SebastienXia!

Thank you for chiming in! The invitation to join the collaborator group has been sent to you. Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers

**Reply by SebastienXia** [reply]
Thanks ^^
